### DISCUSSION TITLE
Decentralized servers

### DISCUSSION THREAD

**Discussion by MadTooth** [stoatchat/discussions#815] [Feature Suggestions]
Your server capacity is not enough, you need to increase the capacity of the servers for a larger number of users, you can buy new servers, or you can be helped by users, with the help of a network of decentralized servers that will distribute the load depending on the load of the servers, or they will be an alternative "home" server, (example of the Matrix network, XMPP), or you will make these servers (your option), if you really want to develop your project for millions users, then you need to solve this global problem, but it's up to you to decide.

**Comment by Shanesan** [comment]
I see this is old, but there are definitely other threads (https://github.com/orgs/revoltchat/discussions/469, https://github.com/orgs/revoltchat/discussions/846, https://github.com/orgs/revoltchat/discussions/249, https://github.com/orgs/revoltchat/discussions/377) you could make this opinion in, just saying.
